src/main/java/virtual_pet/VirtualPet.java

```diff
@@ -0,0 +1,100 @@
+package virtual_pet;
+
+import java.util.Scanner;
+
+public class VirtualPet {
+    Scanner input = new Scanner(System.in);
+    String petName;
+    int hungerLevel = 6, happinessLevel = 6, sleepinessLevel = 6;
+    int angerLevel = 0; // value to track when a pet might leave if it gets neglected
```

Comment:
Lines 7 - 9 should be private to the VirtualPet class and values passed to it through a constructor.